release/nightly_tests/dataset/inference_gce.yaml

```diff
@@ -0,0 +1,28 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west1
+allowed_azs:
+- us-west1-b
+
+max_workers: 999
+
+head_node_type:
+    name: head_node
+    instance_type: n1-standard-64-nvidia-tesla-t4-1
```

Comment:
We actually don't have this type of machine, the list is here: https://github.com/anyscale/product/blob/master/backend/utils/allowed_gpus.yaml